src/gui/FxMixerView.cpp

```diff
@@ -453,7 +453,6 @@ void FxMixerView::moveChannelLeft(int index)
 		delete m_fxChannelViews[i]->m_fader;
 		delete m_fxChannelViews[i]->m_muteBtn;
 		delete m_fxChannelViews[i]->m_soloBtn;
-		delete m_fxChannelViews[i]->m_fxLine;
```

Comment:
@DanWin This should be the fix.

However I can't confirm that we are not leaking memory.